tac/project-updates/2024/2024-Q4-Hyperledger-Fabric.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: 2024 Q4 Hyperledger Fabric
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+Hyperledger Fabric is fairly mature and stable with a v2.5 long-term support (LTS) release and a new v3.0 release.
+There is less churn and fewer commits than in years past, with continued focus on quality, maintenance, and support.
+New features get proposed, approved, and implemented based on a community RFC process.
+Mailing list activity is down a bit compared to prior years.
+PRs and mailing list questions are generally turned around quickly.
+There is significant Discord activity.
+
+# Questions/Issues for the TAC
+
+We would like to shift the project release images from DockerHub to Github Container Registry (ghcr). Note that the project release images are intended for trial use rather than production use, since they are not maintained regularly for security vulnerabilities, etc (production users should work with a vendor for commercially supported images). But still, there are a few concerns we'd like to raise for feedback.
+- Existing deployments and users may have scripts that pull images from DockerHub, for example as part of CI jobs. Would it be acceptable to break these scripts with the move?
+- We could leave all old images on DockerHub, and publish new images to Github Container Registry. Would this be confusing to users?
```

Comment:
IMO this is perfectly fine as long as you have a note on the project's README page (and/or in the official docs) stating that you are doing this so legacy scripts don't break, while warning users that the Dockerhub images are out-of-date and the ghcr dependencies ought to be used instead.